### PR TITLE
Products Page & Menu

### PR DESCRIPTION
**Pull request by eliabejr**

### CODE REVIEW

**Review by guilhermedecampo — APPROVED** [review]

**Review by fakenickels — APPROVED** [review]

**Review by fakenickels — APPROVED** [review]

File: src/pages/product/astrocontrole.js [hunk 1]
```diff
@@ -85,10 +108,44 @@ const Astrocontrole = () => (
...
         </WhatIsThis>
 
-        <Testimonials />
+        <Features>
+          <Title align="center" transform="uppercase" withStripe>
+            What is in the package?
+          </Title>
+
+          <HGroup>
+            <Feature name="Easy to use" icon="https://image.flaticon.com/icons/svg/1642/1642303.svg" />
```
Comment (fakenickels) [inline src/pages/product/astrocontrole.js]: are these free of royalties?

Comment (guilhermedecampo) [reply src/pages/product/astrocontrole.js]: only examples we won't use it.

**Review by fakenickels — APPROVED** [review]

**Review by guilhermedecampo — APPROVED** [review]

File: src/components/Footer.js [hunk 2]
```diff
@@ -71,7 +82,24 @@ const Footer = () => (
             <Medium />
           </a>
         </SocialLinks>
-        <Copyrights>Copyright {new Date().getFullYear()} © Astrocoders Desenvolvimento LTDA. CNPJ 24.896.711/0001-54. Avenida Afonso Vergueiro, 2250, Sala 2, Sorocaba, São Paulo.</Copyrights>
+        <Copyrights>
+          Copyright {new Date().getFullYear()} © Astrocoders Desenvolvimento LTDA. CNPJ 24.896.711/0001-54. Avenida
+          Afonso Vergueiro, 2250, Sala 2, Sorocaba, São Paulo.
+          <CopyrightIcons>
+            Icons made by{' '}
+            <a href="https://www.flaticon.com/authors/eucalyp" title="Eucalyp">
```
Comment (fakenickels) [inline src/components/Footer.js:90]: this should be a styled component also no?

Comment (eliabejr) [reply src/components/Footer.js:90]: not really I think 🤔

**Pull request merged**